config/routes.rb

```diff
@@ -4,7 +4,8 @@
 
   # You can have the root of your site routed with "root"
   # root 'welcome#index'
-
+  resource :contratos
+  get "contratos/index" => 'contratos#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.